### PR TITLE
osfv: initial commit

### PR DESCRIPTION
**Pull request by pietrushnic**

### CODE REVIEW

File: dug_3_osfv.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+.center[<img src="/remark-templates/dasharo-presentation-template/images/dasharo-sygnet-white.svg" width="150px" style="margin-left:-20px">]
+
+???
+
+---
+
+# Agenda
+
+* Introduction to Dasharo Open Sourve Firmware Validation (OSFV)
```
Comment (macpijan) [inline dug_3_osfv.md:13]:
```suggestion
* Introduction to Dasharo Open Source Firmware Validation (OSFV)
```

File: dug_3_osfv.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+???
+
+---
+
+# History
+
+.center.image-99[![](/img/osfv.png)]
+
+* We use OSFV at least since 2018, when we started validate PC Engines releases
+* Using those scripts we over **50k** tests publicly relasing **150+** binaries
```
Comment (macpijan) [inline dug_3_osfv.md:27]: we ... ?

Comment (macpijan) [reply dug_3_osfv.md:27]: releasing

Spellchecker may be rquired

File: dug_3_osfv.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  based on open-source firmware.
+* Initially it was closed source because of assumption that validation provide
```
Comment (macpijan) [inline dug_3_osfv.md:29]:
```suggestion
* Initially it was closed source because of assumption that validation provides
```

File: dug_3_osfv.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+  majority of the value in open-source firmware development.
+
+---
+
+# Dasharo OSVF
+
+.center[https://github.com/Dasharo/open-source-firmware-validation]
+
+* Dasharo Open Sourve Firmware Validation consist of MIT-licensed scripts
+  written in Robot Frameworks with purpose of validation of open-source
```
Comment (macpijan) [inline dug_3_osfv.md:39]:
```suggestion
  written in Robot Frameworks with the purpose of validation of open-source
```

File: dug_3_osfv.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+ 
+.center.image-70[![](/img/osfv_arch.png)]
+
+---
+
+# OSFV v0.2
+
+.center.image-90[![](/img/osfv_warn.png)]
+
+* v0.2 was silent release (no announcements)
```
Comment (macpijan) [inline dug_3_osfv.md:65]:
```suggestion
* v0.2 was a silent release (no announcements)
```

File: dug_3_osfv.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+    temperature, power cycle testing.
+  - **Dashro Security Features Testing**: UEFI Secure Boot, measured boot, TPM,
+    verified boot, TCG OPAL, ME, DMA protection and more.
+* To maximize use of Dasharo OSFV you need dedicated infrastructure.
+
+---
+
+# OSFV Lab
+ 
+.center.image-70[![](/img/osfv_arch.png)]
```
Comment (macpijan) [inline dug_3_osfv.md:57]: We may mention PiKVM which was not there while creating this graphic a few years ago

File: dug_3_osfv.md (hunk 2 above)
Comment (pietrushnic) [reply dug_3_osfv.md:27]: I cannot afford your rates for spell checking to quote the guru, just leave one comment. I will fix those if/when I will find time before DUG

**Pull request merged**